### PR TITLE
Make the comments great again

### LINKED ISSUE #82: Make the comments great again

### PR DESCRIPTION
**Pull request by jstnceronio**
closes #82

### CODE REVIEW

File: marketplace/src/app/pages/post/single-view/post-view/post-view.component.html [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
-            <textarea cols="40" rows="5" formControlName="body"  class="input" type="text" placeholder="Kommentar abgeben"></textarea>
-            <br>
+        <div class="comment-action">
+            <textarea cols="40" rows="1" formControlName="body" class="input" type="text" placeholder="Kommentar abgeben"></textarea>
             <div class="actions">
                 <button (click)="postComment()" class="btn-submit" type="submit">Veröffentlichen</button>
             </div>
         </div>
     </form>
+    <div class="mt-4" *ngFor="let comment of comments | async">
```
Comment (Akjo03) [inline marketplace/src/app/pages/post/single-view/post-view/post-view.component.html:20]: this is kind of stupid i know but move this into the scss file (clean code)

**Review by RoadJDK — APPROVED** [review]
<3

File: marketplace/src/app/pages/post/single-view/post-view/post-view.component.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+* {
+    @apply font-montserrat;
```
Comment (Akjo03) [inline marketplace/src/app/pages/post/single-view/post-view/post-view.component.scss:2]: shouldn't this be on the whole app?

**Review by Akjo03 — APPROVED** [review]
nice changes but some minor clean code things

File: marketplace/src/app/pages/post/single-view/post-view/post-view.component.scss (hunk 2 above)
Comment (jstnceronio) [reply marketplace/src/app/pages/post/single-view/post-view/post-view.component.scss:2]: out of scope for this PR

Comment (Akjo03) [reply marketplace/src/app/pages/post/single-view/post-view/post-view.component.scss:2]: can be solved in other issue

File: marketplace/src/app/pages/post/single-view/post-view/post-view.component.html (hunk 1 above)
Comment (Akjo03) [reply marketplace/src/app/pages/post/single-view/post-view/post-view.component.html:20]: can not be done beacuse ngFor maybe shit

**Pull request merged**